yt_dlp/extractor/vimp.py

```diff
@@ -0,0 +1,152 @@
+from .common import InfoExtractor
+from ..utils import ExtractorError
+
+
+class VimpIE(InfoExtractor):
```

Comment:
Changing extractor name breaks `--download-archive`. So it should not be done unless absolutely necessary. You can set the `IE_NAME` and `IE_DESC` instead to control the name that users see

Reply:
So it will always be the `VideocampusSachsen` extractor? :D